app/services/katello/pulp3/repository_mirror.rb

```diff
@@ -169,7 +169,7 @@ def refresh_distributions(options = {})
         if (distro = repo_service.lookup_distributions(base_path: path).first) ||
           (distro = repo_service.lookup_distributions(name: "#{backend_object_name}").first)
           # update dist
-          dist_options = dist_options.except(:name, :base_path)
+          dist_options = dist_options.except(:name)
```

Comment:
could you explain why this change is needed?

Reply:
Without it, the `base_path` was not getting updated on the smart proxies.

Reply:
ohhh yes, that makes sense.  I think i was reading it backwards